### ISSUE TITLE
CCLabel: robust handling of font filenames

### ISSUE THREAD

**Issue by rtabbara** [mono/CocosSharp#75]
Currently, when specifying a font file for a new <code>CCLabel</code> a user has to do something like:

<pre>
<code>
var label = new CCLabel("string", "fonts/myFont.ttf", 48)
</code>
</pre>

where fonts is a subdirectory of the root Content folder. To make it consistent with how other assets are loaded we should:

* optionally require a user to specify the .ttf suffix
* respect the ContentSearchPath specified in the app delegate so that a user can simply specify the filename

In other words, allow initialisation of a <code>CCLabel</code> to be:

<pre>
<code>
var label = new CCLabel("string", "myFont", 48)
</code>
</pre>

**Comment by kjpou1** [comment]
Closed via the previous two commits associated with this issue.  The .ttf suffix is still required for all .ttf fonts files that are to be loaded.

If the ContentSearchPaths are setup correctly for example:

            application.ContentSearchPaths = new List<string>() { "HD" };
            application.ContentSearchResolutionOrder = new List<string>() { "images", "fonts" };


Then the 

    var label = new CCLabel("string", "fonts/myFont.ttf", 48)

Can be replaced with the following:

    var label = new CCLabel("string", "myFont.ttf", 48)
